packages/margined_utils/src/scenarios/mod.rs

```diff
@@ -772,29 +756,15 @@ impl ShutdownScenario {
         let engine_id = router.store_code(contract_engine());
         let pricefeed_id = router.store_code(contract_mock_pricefeed());
 
-        let insurance_fund_addr = router
-            .instantiate_contract(
-                insurance_fund_id,
-                owner.clone(),
-                &InsuranceFundInstantiateMsg {
-                    beneficiary: "owner".to_string(),
-                },
-                &[],
-                "insurance_fund",
-                None,
-            )
-            .unwrap();
-        let insurance_fund = InsuranceFundController(insurance_fund_addr.clone());
-
         // set up margined engine contract
         let engine_addr = router
             .instantiate_contract(
                 engine_id,
                 owner.clone(),
                 &InstantiateMsg {
                     pauser: owner.to_string(),
-                    insurance_fund: insurance_fund.addr().to_string(),
-                    fee_pool: insurance_fund.addr().to_string(),
+                    insurance_fund: "owner".to_string(),
+                    fee_pool: "owner".to_string(),
```

Comment:
If I was going to nitpick, would rename to "insurance_fund" and "fee_pool" resp.